### ISSUE TITLE
Getting an error when i click  "Buy" on a buy order

### ISSUE THREAD

**Issue by Jared-Dahlke** [bisq-network/bisq#6635]
### Description

When I click on an order to buy it takes me to the details page, it pauses for a few seconds , then i see this error:
`Warning
An error occurred at task: ProcessOfferAvailabilityResponse
Cannot take offer because snapshot height does not match. Probably your DAO data are not synced.`

#### Version
`v1.9.9`

### Screenshots
![image](https://user-images.githubusercontent.com/32936978/230225411-c1a60738-29d5-445c-9120-c5d76183e299.png)

#### Device or machine
Macbook pro

#### Additional info

I just started seeing this bug. I've been using bisq for a couple years now. I tried deleting/resyncing SVP file but still have this problem.

**Comment by MwithM** [comment]
Have you tried if a DAO resync is needed at _DAO - Network status_?

**Comment by Jared-Dahlke** [comment]
I was able to take a different buy order successfully yesterday, i think it may have just been that person's problem? 

I took like 4 buy orders yesterday. I was getting this error on a couple different people's orders.

Could it have been a problem with their dao status or their system?


Here is a screenshot of my status:
![image](https://user-images.githubusercontent.com/32936978/230399865-bd69d235-46a0-4889-be98-371eba3b32a0.png)
